region/norfair/west.json

```diff
@@ -765,17 +803,60 @@
           "to": [
             {
               "id": 2,
-              "requires": [ "Morph" ],
+              "requires": [
+                {"or": [
+                  "heatProof",
+                  "canHeatRun"
+                ]},
+                "Morph",
+                {"or": [
+                  {"cost": [
+                    {"heatFrames": 300},
+                    [
+                      {"enemyDamage": {
+                        "enemy": 99,
+                        "hits": 1
+                      }}
+                    ]
+                  ]},
+                  {"cost": [
+                    {"heatFrames": 300},
+                    [
+                      {"ammo": {
```

Comment:
Do you think it's worth also looking at alternatives with wave beam?

Reply:
Absolutely. Tomorow :)

Reply:
Difference is less than 50f both from top and bottom door. Leaving them unchanged.

Reply:
Alrighty then